plutus-tx-plugin/test/Budget/find.budget.golden

```diff
@@ -1 +1 @@
-ExBudget {exBudgetCPU = ExCPU 49371287, exBudgetMemory = ExMemory 161020}
\ No newline at end of file
+ExBudget {exBudgetCPU = ExCPU 49758336, exBudgetMemory = ExMemory 162320}
```

Comment:
I suspect these test outputs are flip-flopping, you might just want to revert them.